Desktop Version

AbhishekkM05 opened the pull request:
Add Mobile navigation menu using Javascript.
Add Speaker section using javascript.
Complete Mobile Version 
Complete desktop version 
Update Read Me 
Here is my Loom video link https://www.loom.com/share/b8b3db38649d40a19bffdba1166353b6

MaganaAsiati reviewed (changes requested):
# STATUS: Required Changes ♻️♻️♻️ 
## Hi @AbhishekkM05 , I'm @MaganaAsiati, your reviewer for this PR  :wave: 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

![so-close-gif-4](https://user-images.githubusercontent.com/95297251/191952218-b7185270-c02b-4e4d-89af-eae77fcd8bea.gif)

## Highlights

- Descriptive README file  :heavy_check_mark:👨🏻‍💻
- Great work using the right GitHub Flow.:heavy_check_mark:
- No Linter errors :heavy_check_mark:

## Required Changes ♻️

- [ ]  kindly make sure this section looks like it shows in the design provided, I can suggest you remove the height for your  description class :+1: 
https://github.com/AbhishekkM05/Global-Innovation-Summit-/blob/1770dafcb22e113e7b8d40e3545fa54595d8598b/about.html#L56-L58
 
## your design 
![image](https://user-images.githubusercontent.com/95297251/198573773-ac39e16d-cd94-41e7-8e90-ddbbe76239fa.png)

## Design provide 

![image](https://user-images.githubusercontent.com/95297251/198573849-c0673c60-6a9a-4f83-be8a-88127c783f01.png)

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._
- [ ] [OPTIONAL] Please make sure to make your commit messages more descriptive in the future. :heavy_check_mark:👨🏻‍💻
- [ ] [OPTIONAL] kindly  Add a meaningful description to your pull request in the future which gives more information about what you implement in this milestone :heavy_check_mark:👨🏻‍💻


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

AbhishekkM05 commented on index.html:
```diff
@@ -88,16 +86,13 @@ <h3>GIS Ingite</h3>
-      <button class="join" type="submit">Join MMF NOW</button>
+      <button class="join" type="submit">Join GIS NOW</button>
     </section>
      <section class="speakers-sec" id="Geusts">
```
it is optional only !!

AbhishekkM05 commented on media.css:
```diff
@@ -0,0 +1,155 @@
+    height: auto;
+  }
+
+  .title {
+    font-size: 4rem;
+    line-height: 4rem;
+    font-weight: 700;
+    text-transform: uppercase;
+  }
```
it is working fine on my system in mobile version

AbhishekkM05 replied:
i have used different backgroound as have many people

AbhishekkM05 commented:
update the required changes.
removed the height as asked.
removed the margins as asked.
create border box for home text. 
fix the image issues.
gave left margin to speaker section as asked
Image used and title is personalized .
Less more button is optional in design .

youmari reviewed (approved):
Hi @AbhishekkM05 , 

In my opinion, your project is ready for the presentation!  There is nothing else to say other than... it's time to merge it :shipit: 

![](https://media.giphy.com/media/g9582DNuQppxC/giphy-downsized-large.gif)
Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
